caikit/core/module.py

```diff
@@ -510,7 +729,7 @@ def _extract_pred_set(self, dataset, *args, preprocess_func=None, **kwargs):
         )
 
     @staticmethod
-    def load_evaluation_dataset(dataset_path):
+    def _load_evaluation_dataset(dataset_path):
```

Comment:
@alex-jw-brooks I'm pretty sure you owned this originally. To the best of my knowledge, this function is an implementation detail that's only used in the default implementation of `evaluate_quality`. Is that right?

Reply:
Hey @gabe-l-hart! As far as I'm aware, that is correct